tests/app/service/send_notification/test_send_one_off_notification.py

```diff
@@ -160,7 +159,7 @@ def test_send_one_off_notification_honors_priority(
 ):
     service = create_service()
     template = create_template(service=service)
-    template.process_type = PRIORITY
+    template.process_type = TemplateProcessType.PRIORITY
```

Comment:
Might be something we need to remove or skip if this is testing priority queues.